pulp_rpm/app/serializers.py

```diff
@@ -633,7 +638,13 @@ def validate(self, data):
         if hasattr(self, 'initial_data'):
             validate_unknown_fields(self.initial_data, self.fields)
 
+        if 'criteria' in data and 'content' in data:
+            raise serializers.ValidationError(
+                _("Criteria and content fields cannot both be set.")
+            )
```

Comment:
I'm debating about changing this? I think Katello will want to specify hrefs and criteria.